pkg/node/node.go

```diff
@@ -684,21 +685,17 @@ func NewBee(interrupt chan struct{}, sysInterrupt chan os.Signal, addr string, p
 		eventListener               postage.Listener
 	)
 
-	var postageSyncStart uint64 = 0
-
 	chainCfg, found := config.GetByChainID(chainID)
-	postageStampContractAddress, startBlock := chainCfg.PostageStampAddress, chainCfg.PostageStampStartBlock
+	postageStampContractAddress, postageSyncStart := chainCfg.PostageStampAddress, chainCfg.PostageStampStartBlock
 	if o.PostageContractAddress != "" {
 		if !common.IsHexAddress(o.PostageContractAddress) {
 			return nil, errors.New("malformed postage stamp address")
 		}
 		postageStampContractAddress = common.HexToAddress(o.PostageContractAddress)
+		postageSyncStart = o.PostageContractStartBlock
```

Comment:
Would be nice to have a check if the `o.PostageContractStartBlock > 0` before assigning it.

Reply:
I think it would be good to even return an error if that happens